pages/common/glow.md

```diff
@@ -0,0 +1,16 @@
+# glow
+
+> Render Markdown in the terminal.
+> More infomration: <https://github.com/charmbracelet/glow>.
+
+- Run glow and select a file to view:
+
+`glow`
+
+- Render a Markdown file to the terminal:
+
+`glow {{FILE}}`
+
+- View a Markdown file (paginated):
```

Comment:
Maybe this is better:
```suggestion
- Open a markdown file using a pager:
```

Reply:
What do you think of the change in 7dc2f82db1aa60e30571643c063aaece99f85846?

Reply:
Looks good.